### PR TITLE
Engagements: creating task

### PR DESCRIPTION
**Pull request by VSydor**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
   <groupId>com.impactupgrade.integration</groupId>
   <artifactId>hubspot-java-client</artifactId>
-  <version>2.1.0-SNAPSHOT</version>
+  <version>2.1.1-SNAPSHOT</version>
```
Comment (brmeyer) [inline pom.xml]: Safe to keep 2.1.0-SNAPSHOT for now -- still iterating on this until LJI settles down.

File: src/main/java/com/impactupgrade/integration/hubspot/v1/model/EngagementEmailMetadata.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package com.impactupgrade.integration.hubspot.v1.model;
+
+public class EngagementEmailMetadata extends EngagementMetadata {
```
Comment (brmeyer) [inline src/main/java/com/impactupgrade/integration/hubspot/v1/model/EngagementEmailMetadata.java:3]: Oh man, thank you for doing this! We have a ticket to do this exactly based off Mailchimp events...

**Review by brmeyer — COMMENTED** [review]
LGTM! One comment about the version, but good otherwise!

File: pom.xml (hunk 1 above)
Comment (VSydor) [reply pom.xml]: Updating

**Pull request merged**